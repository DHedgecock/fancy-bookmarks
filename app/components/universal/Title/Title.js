import styled from 'styled-components'

const Title = styled.h1`
  font-family: ${({ theme }) => theme.fontFamilySansSerif};
  font-size: ${({ theme }) => theme.h1FontSize};
  text-align: center;
`

export default Title
