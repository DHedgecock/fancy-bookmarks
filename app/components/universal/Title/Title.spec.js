import React from 'react'
import { shallow } from 'enzyme'

import Title from './Title'

test('renders', () => {
  const actual = shallow(<Title>Test</Title>)
  // Getting testing running ¯\_(ツ)_/¯
  expect(actual).toBeTruthy()
})

// Snapshots
// ---------------------------------------------------------------------------
test('renders h1', () => {
  const tree = shallow(<Title>Test</Title>)
  expect(tree).toMatchSnapshot()
})
