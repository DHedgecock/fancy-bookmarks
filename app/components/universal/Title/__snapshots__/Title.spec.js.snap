// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders h1 1`] = `
<h1
  className="sc-bdVaJa eUErKY"
>
  Test
</h1>
`;
